stubs/defaultConfig.stub.js

```diff
@@ -1,3 +1,5 @@
+const _ = require('lodash')
```

Comment:
You're not using `lodash` in this file, so this import can be deleted, no?

Reply:
Yep good catch, thanks!